src/components/overlays/StartOverlay.module.scss

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
```

Comment:
You can avoid adding this rule by changing `@import-normalize;` to `@import-sanitize;` in main **index.scss**.

I remember it was in docs but for some reason it was changed to **normalize**. I checked **sanitize** still works.